_common/GFMAlerts.php

```diff
@@ -16,38 +16,34 @@
 
 class GFMAlerts extends Parsedown
 {
-    public $VERSION = '1.0';
-    
-    public function __construct()
-    {
-       $this->BlockTypes['>'][] = 'Alert';
```

Comment:
This was overwriting the default `blockQuote` definition, unnecessarily, requiring a `blockAlert` function. Instead, we just override `blockQuote()` and add handling when an alert pattern is found.